Makefile

```diff
@@ -201,6 +203,8 @@ uninstall-man-rdw:
 
 uninstall: uninstall-tlp uninstall-rdw
 
+uninstall-man: uninstall-man uninstall-man-rdw
```

Comment:
I think this should read:

    uninstall-man: uninstall-man-tlp uninstall-man-rdw

Reply:
You are right, updated.